packages/notifier/src/publish-kit.js

```diff
@@ -316,20 +316,21 @@ const provideDurablePublishKitEphemeralData = (state, facets) => {
  * Extends the sequence of results.
  *
  * @param {{state: DurablePublishKitState, facets: PublishKit<*>}} context
- * @param {boolean} done
  * @param {any} value
- * @param {any} [rejection]
+ * @param {DurablePublishKitState["status"]} [targetStatus]
```

Comment:
🙌 types

Reply:
This is a type syntax I have not seen before. What's the difference between
```js
DurablePublishKitState["status"]
```
vs
```js
DurablePublishKitState.status
```
or
```js
DurablePublishKitState['status']
```
?

Reply:
The two quotes are just stylistic. Our Prettier config is single quotes but it doesn't reach into jsdoc.

The `.` is reserved for referencing a namespace.

https://www.typescriptlang.org/play?#code/C4TwDgpgBAysCGxoF4oG8BQUoDMD2eAXFAHYCuAtgEYQBOANFlFfLcQM7C0CWJA5gG4MAXwwYAxnhKco7XnwA2EYnEQQA2gHJ8eTQF0oqAExDJ04FAAmeMlSUqESdQCIdzg8dNSZJeBQjsYPDiyrCOEAB0OoZQJhhAA